x-pack/test_serverless/functional/test_suites/observability/discover/context_awareness/index.ts

```diff
@@ -35,6 +35,9 @@ export default function ({ getService, getPageObjects, loadTestFile }: FtrProvid
       await PageObjects.timePicker.resetDefaultAbsoluteRangeViaUiSettings();
     });
 
+    loadTestFile(require.resolve('./_get_row_indicator_provider'));
+    loadTestFile(require.resolve('./_get_doc_viewer'));
+    loadTestFile(require.resolve('./_get_cell_renderers'));
```

Comment:
We could also copy and adjust telemetry / context tests from `test/functional/apps/discover/context_awareness/_telemetry.ts` to check what profile names are active and sent to EBT.